lib/src/remote_devtools_middleware.dart

```diff
@@ -27,36 +27,35 @@ class RemoteDevToolsMiddleware<State> extends MiddlewareClass<State> {
   ///
   ///
   final String _host;
-  SocketClusterWrapper socket;
-  Store<State> store;
-  String _channel;
+  late SocketClusterWrapper socket;
+  Store<State>? store;
+  late String _channel;
   RemoteDevToolsStatus status = RemoteDevToolsStatus.notConnected;
 
   /// The function used to decode actions. If not specifies, defaults to [NopActionDecoder]
-  ActionDecoder actionDecoder;
+  late ActionDecoder /*late*/ actionDecoder;
```

Comment:
I assume the /*late*/ is a Hint marker that didn't get updated?  (I saw this earlier and was going to comment after I'd experimented with the PR but missed my chance :-)

Reply:
Ah oops, yes. The migration tool _really didn't_ wants respect the hint so I did that migration manually. I'll remove it.